tests/test_v2.py

```diff
@@ -39,3 +50,31 @@ def test_write_netrc_file(tmp_path):
     main.write_netrc_file('already_there', 'this call should do nothing')
     with open(output_file, 'r') as f:
         assert f.read() == 'machine urs.earthdata.nasa.gov login foo password bar'
+
+
+def test_upload_file_to_s3(tmp_path, s3_stubber):
+    expected_params = {
+        'Body': ANY,
+        'Bucket': 'myBucket',
+        'Key': 'myFile.zip',
+        'ContentType': 'application/zip',
+    }
+    s3_stubber.add_response(method='put_object', expected_params=expected_params, service_response={})
+
+    os.chdir(tmp_path)
+    Path('myFile.zip').touch()
```

Comment:
You don't need to do the `os.chdir` bit it looks like, or create a new `Path`. I think it'd work better to do something like:

```python
upload_file = tmp_path / 'myFile.zip'
upload_file.touch()
 main.upload_file_to_s3(upload_file, 'myBucket')
```
Possible need to wrap `upload_file` in `str` if Boto3 doesn't understand pathlib paths